app/views/admin/calendars/_form.html.erb

```diff
@@ -4,7 +4,10 @@
       class: "btn btn-sm btn-primary" unless @calendar.new_record? %>
 </p>
 
-<%= render 'importer_overview', calendar: @calendar %>
+<%# don't show importer stats if it's never run %>
```

Comment:
why not?

the partial only renders if the calendar has a model behind it and if the importer has never run then it should just be in 'idle' mode.

Reply:
if it's not imported yet then `last_import_at` is `nil` and all the date related code goes wonky